pkg/machines/components/storagePools/createStoragePoolDialog.jsx

```diff
@@ -71,13 +71,15 @@ const StoragePoolNameRow = ({ onValueChanged, dialogValues }) => {
     );
 };
 
-const StoragePoolTypeRow = ({ onValueChanged, dialogValues }) => {
-    const poolTypes = [
+const StoragePoolTypeRow = ({ onValueChanged, dialogValues, libvirtVersion }) => {
+    let poolTypes = [
         { type: 'dir', detail: _("Filesystem Directory") },
         { type: 'netfs', detail:_("Network File System") },
         { type: 'iscsi', detail: _("iSCSI Target") },
-        { type: 'iscsi-direct', detail: _("iSCSI direct Target") }
     ];
+    // iscsi-direct exists since 4.7.0
+    if (libvirtVersion && libvirtVersion >= 4007000)
```

Comment:
@Antique does libvirt provide any better way to check if a driver is available?

In this case I want to check support for iscsi-direct storage driver, which I know got introduced in 4.7.0, but it's still cumbersome to have hardcoded checks for libvirt version, but I was not able to find an API to query driver support.

Reply:
There is a new API virConnectGetStoragePoolCapabilities introduced in libvirt 5.2.0 so in this case it will not help you and you will probably have to keep the version check, but that API could be useful for future work.

Reply:
Thanks for the feedback!